### DISCUSSION TITLE
How much do I need to contribute to get to participate in octernship?

### DISCUSSION THREAD

**Discussion by gautam-andani** [community/community#50680] [GitHub Education]
I am new to using github I don't know much about open source. I've read blog on https://opensource.guide/how-to-contribute/ but am still unsure where and what exactly to contibute.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @gautam-andani, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
